seed/client/src/main/scala/example/seed/client/process/ProtoPeopleServiceClient.scala

```diff
@@ -54,7 +54,7 @@ object ProtoPeopleServiceClient {
           _      <- L.info(s"")
           result <- client.getPerson(PeopleRequest(name))
           _      <- L.info(s"$serviceName - Request: $name - Result: $result")
-        } yield result.person
+        } yield result.person.get
```

Comment:
^^ `Option.get`

Reply:
Is this still pending, right?

Reply:
oh what?!  I thought for sure I got this one, ugh.  Sorry!  I got the one lower in the file 🤦